docs/DEVELOPING.md

```diff
@@ -737,6 +737,121 @@ $ make ftest NAME=mssql
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.8_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+Confluence versions 7 or later are compatible with Elastic connector frameworks.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Confluence details](#gather-confluence-details)
+2. [Configure Confluence connector](#configure-confluence-connector)
+
+#### Gather Confluence details
+
+Collect the information that is required to connect to your Confluence instance:
+
+- The server host url where Confluence is hosted.
+- Username for the Confluence server or service account for the Confluence cloud.
+- Password for the Confluence server or API token for the Confluence cloud.
+- SSL certificate for a secure connection
+
+#### Configure Confluence connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `is_cloud`
+
+Flag to determine the Confluence platform type. `True` if Confluence cloud and `False` if Confluence server. Default value is `False`.
+
+##### `username`
+
+The username of the account for Confluence server. Default value is `admin`.
+
+##### `password`
+
+The password of the account to be used for the Confluence server. Default value is `abc@123`.
+
+##### `service_account_id`
+
+The service account for the Confluence cloud. Default value is `me@example.com`.
+
+##### `api_token`
+
+The API token to authenticate with Confluence cloud. Default value is `abc#123`.
+
+##### `host_url`
+
+The server host url where the Confluence is hosted. Default value is `http://127.0.0.1:5000`. Examples:
+
+  - `https://192.158.1.38:8080/`
+  - `https://test_user.atlassian.net/`
+
+##### `ssl_enabled`
+
+Whether SSL verification will be enabled. Default value is `False`.
+
+##### `ssl_ca`
+
+Content of SSL certificate. Note: In case of ssl_enabled is `False`, keep `ssl_ca` field empty. Example certificate:
+
+  - ```
+    -----BEGIN CERTIFICATE-----
+    MIID+jCCAuKgAwIBAgIGAJJMzlxLMA0GCSqGSIb3DQEBCwUAMHoxCzAJBgNVBAYT
+    ...
+    7RhLQyWn2u00L7/9Omw=
+    -----END CERTIFICATE-----
+    ```
+
+##### `retry_count`
+
+The number of retry attempts after failed request to Confluence. Default value is `3`.
+
+##### `concurrent_downloads`
+
+The number of concurrent downloads for fetching the attachment content. This speeds up the content extraction of attachments. Defaults to `50`.
+
+##### `enable_content_extraction`
+
+Whether the connector should extract the content from Confluence attachment. Default value is `True` i.e. the connector will try to extract file contents.
+
```

Comment:
@danajuratoni the change requires using new set of API endpoints for the connector since currently pages, blogposts and attachments are fetched independent of the space.

Since this is a scope change, can you please create an enhancement issue and we'll take it once the present one is merged on main.

Reply:
Sure, this can be added as an enhancement. Created https://github.com/elastic/connectors-python/issues/659